openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -413,7 +413,7 @@ There are two possible ways to request issuance of a specific Credential type in
 The request parameter `authorization_details` defined in Section 2 of [@!RFC9396] MUST be used to convey the details about the Credentials the Wallet wants to obtain. This specification introduces a new authorization details type `openid_credential` and defines the following parameters to be used with this authorization details type:
 
 * `type`: REQUIRED. String that determines the authorization details type. MUST be set to `openid_credential` for the purpose of this specification.
-* `credentials_supported_id`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credentials_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credentials_supported` map conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
+* `credential_configuration_id`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

Comment:
```suggestion
* `credential_configuration_id`: REQUIRED. A string that uniquely identifies the Credential defined in the `credential_configurations_supported` map within the Credential Issuer Metadata, as detailed in (#credential-issuer-parameters). The corresponding object in the `credential_configurations_supported` map provides specific information, such as format, for the issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

Reply:
please do the bikeshedding in PR https://github.com/openid/OpenID4VCI/pull/155. this PR is strictly meant to change the parameter names only.